src/Compiler/Microsoft.NET.Sdk.Razor.SourceGenerators/RazorSourceGenerator.RazorProviders.cs

```diff
@@ -45,6 +45,7 @@ private static (RazorSourceGenerationOptions?, Diagnostic?) ComputeRazorSourceGe
             globalOptions.TryGetValue("build_property.RootNamespace", out var rootNamespace);
             globalOptions.TryGetValue("build_property.SupportLocalizedComponentNames", out var supportLocalizedComponentNames);
             globalOptions.TryGetValue("build_property.GenerateRazorMetadataSourceChecksumAttributes", out var generateMetadataSourceChecksumAttributes);
+            globalOptions.TryGetValue("build_property.TestingSuppressUniqueIds", out var suppressUniqueIds);
```

Comment:
Is it intentional to call this `TestingSuppressUniqueIds` different from the property name `TestSuppressUniqueIds`?